solvis/inversion_solution/composite_solution.py

```diff
@@ -43,7 +43,8 @@ def rupture_rates(self) -> pd.DataFrame:
         """
         Calculate (and cache) the rates.
 
-        :return: a gpd.GeoDataFrame
+        Returns:
+            a `gpd.GeoDataFrame`
```

Comment:
- is it returning a geopanda or plain pandas dataframe??

Reply:
That's a good question that I don't currently know the answer to. I haven't yet had the opportunity to dig into how these operate in practice - just reformatting the docstrings.

As you might expect, a `geopandas.GeoDataFrame` appears to be a subclass of a `pandas.DataFrame`. I'll see if I can work out from the tests if a tighter typing is more appropriate.

Reply:
pandas it is - thanksyeah I thought about something like that, but decided to leave it for the next round. There